src/components/specialized/Achievements/AchievementsChain/container.js

```diff
@@ -8,13 +8,13 @@ const mapStateToProps = (state, { achievementsChain }) => ({
   canPerformActions: R.allPass([
     S.facebook.login.isLogged,
     S.facebook.registration.isRegistered,
-    S.wallets.qtum.isReady
+    S.wallets.decent.isReady
   ])(state),
   currentAchievement: S.achievement.currentFrom(achievementsChain),
   pastAchievements: S.achievement.pastFrom(achievementsChain),
   userID: S.facebook.login.userID(state),
-  walletAddress: S.wallets.qtum.address(state),
-  walletBalance: S.wallets.qtum.balance(state)
```

Comment:
We need to still keep code for QTUM.
I think if Achivements are specifice for each blockchain, we need to create Achivements components inside a WALLET directory.

For ex:
```
Qtum/
    Achievements/
    Wallet/
Decent/
   Achievements/
   Wallet/
```